.jenkins/pytorch/test.sh

```diff
@@ -562,6 +568,8 @@ elif [[ "${BUILD_ENVIRONMENT}" == *distributed* || "${JOB_BASE_NAME}" == *distri
   test_rpc
 elif [[ "${TEST_CONFIG}" = docs_test ]]; then
   test_docs_test
+elif [[ "${BUILD_ENVIRONMENT}" == *linux-xenial-cuda11.3-py3.6-gcc7* ]]; then
```

Comment:
Could we base this on TEST_CONFIG being fx2trt instead? Something like 
```suggestion
elif [[ "${TEST_CONFIG}" == fx2trt ]]; then
```